portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro

```diff
@@ -136,6 +136,60 @@ definition {
 			ACSettings.deleteDataSource();
 		}
 	}
+	
+	macro createRequestLogByEmail {
+		var key_fieldType = "${fieldType}";
+
+		ACNavigation.goToSettings();
```

Comment:
Hi, @AfonsoHSantos. I think this navigation is better to stay out of the macro, because there may be test cases that a person is already in settings for having done another step and then to use this macro, they would have to leave the settings and then use them. What do you think?

Reply:
fixed